Examples/Io/Root/src/RootBFieldWriter.cpp

```diff
@@ -224,48 +225,37 @@ void RootBFieldWriter::run(const Config& config,
 
         nBinsR = nBins.at(0);
         nBinsZ = nBins.at(1);
-        nBinsPhi = 100.;
 
       } else {
-        std::ostringstream errorMsg;
-        errorMsg << "BField has wrong dimension. The dimension needs to be "
-                    "either 2 (r,z,Br,Bz) or 3(x,y,z,Bx,By,Bz) in order to be "
-                    "written out by this writer.";
-        throw std::invalid_argument(errorMsg.str());
+        throw std::invalid_argument(
+            "BField has wrong dimension. The dimension needs to be "
+            "either 2 (r,z,Br,Bz) or 3(x,y,z,Bx,By,Bz) in order to be "
+            "written out by this writer.");
       }
     }
-    double minPhi = -M_PI;
-    stepR = fabs(minR - maxR) / nBinsR;
-    stepZ = fabs(minZ - maxZ) / nBinsZ;
-
-    nBinsR += 1;
-    nBinsZ += 1;
-
-    double stepPhi = (2 * M_PI) / nBinsPhi;
-
-    for (size_t i = 0; i < nBinsPhi; i++) {
-      double phi = minPhi + i * stepPhi;
-      for (size_t k = 0; k < nBinsZ; k++) {
-        double raw_z = minZ + k * stepZ;
-        for (size_t j = 0; j < nBinsR; j++) {
-          double raw_r = minR + j * stepR;
-          Acts::Vector3 position(raw_r * cos(phi), raw_r * sin(phi), raw_z);
-          ACTS_VERBOSE("Requesting position: " << position.transpose());
-          auto bField = config.bField->getFieldUnchecked(position);
-          z = raw_z / Acts::UnitConstants::mm;
-          r = raw_r / Acts::UnitConstants::mm;
-          Bz = bField.z() / Acts::UnitConstants::T;
-          Br = VectorHelpers::perp(bField) / Acts::UnitConstants::T;
-          outputTree->Fill();
-        }
-      }
-    }  // for
+
+    stepR = fabs(minR - maxR) / (nBinsR - 1);
+    stepZ = fabs(minZ - maxZ) / (nBinsZ - 1);
```

Comment:
switch to `std::abs`?

Reply:
Same questions as above. I'll resolve this one.